app/views/application/_nav.html.slim

```diff
@@ -8,13 +8,13 @@ nav.navbar.navbar-default.navbar-static-top.top-nav-collapse
             .user-menu
               input.user-menu-trigger type="checkbox"
               .user-avatar= render "avatar_svg"
-              span= current_publisher.name
+              span#current-user-name= current_publisher.name
               .user-lock= render two_factor_enabled?(current_publisher) ? "locked_svg" : "unlocked_svg"
               .user-arrow= render "arrow_svg"
               .user-dropdown
                 = link_to home_publishers_path, class: "user-info" do
                   .user-avatar= render "avatar_svg"
-                  .user-name= current_publisher.name
+                  .user-name#user-name-dd= current_publisher.name
```

Comment:
what is `dd`?

Reply:
ic it might be "dropdown", now i understand.
i think it'd be preferable to spell out "dropdown" to make it super clear for future devs.